### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by elixir14082002**
<hr>

## Description 📜

I have added the topic and the link for the HTML Canvas in the readme file.

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [x] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [x] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by elixir14082002** [comment]
@Ayan-thecodeking

File: Web_Development/FrontEnd/ReadMe.md [hunk 1]
```diff
@@ -18,6 +18,7 @@
 - [HTML Links and Images](Html_Anchor_img_tag.md)
 - [HTML Tables](./HTML_Tables)
 - HTML Lists
+- [HTML Canvas](https://github.com/girlscript/winter-of-contributing/blob/61250d55f0f3adab4f31b9472e9188f907ad9f11/Web_Development/FrontEnd/HTML_Canvas/Canvas.md)
```
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/ReadMe.md:21]: Please change the link in a similar format as others. 
`
[HTML Canvas](./Web_Development/FrontEnd/HTML_Canvas/Canvas.md)
`

**Comment by elixir14082002** [comment]
@Aayushi-Mittal I have made the necessary changes. Kindly review!

File: Web_Development/FrontEnd/ReadMe.md [hunk 2]
```diff
@@ -18,7 +18,7 @@
 - [HTML Links and Images](Html_Anchor_img_tag.md)
 - [HTML Tables](./HTML_Tables)
 - HTML Lists
-- [HTML Canvas](https://github.com/girlscript/winter-of-contributing/blob/61250d55f0f3adab4f31b9472e9188f907ad9f11/Web_Development/FrontEnd/HTML_Canvas/Canvas.md)
+- [HTML Canvas](./HTML_Canvas)
```
Comment (Aayushi-Mittal) [inline Web_Development/FrontEnd/ReadMe.md:21]: The link is redirecting to 404. Please change it to `./Canvas.md`

**Comment by elixir14082002** [comment]
@Aayushi-Mittal I have changed the link. Kindly review!

**Review by Aayushi-Mittal — APPROVED** [review]
Looks Perfect now! @Ayan-thecodeking Please merge this PR.

**Pull request merged**